centaur/src/main/resources/standardTestCases/retry_with_more_memory/retry_with_more_memory.wdl

```diff
@@ -12,7 +12,6 @@ task imitate_oom_error {
   runtime {
     docker: "python:latest"
     memory: "1 GB"
-    continueOnReturnCode: true
     maxRetries: 2
     backend: "Papiv2"
```

Comment:
Is this backend value valid? I see this in tests:

Unrecognized runtime attribute keys: backend

Reply:
Yes, but because it applies outside of the backend (eg PAPI, AWS, Azure) code, the backend classes don't recognize it.